File: examples/python/automations/links/get.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.links.get(
    link_id="lnk_Xf15nXFpo7mwfGT1aSYo4",
)

print(automation.model_dump_json(indent=2))

File: examples/node/automations/links/get.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

config({ path: '../../../.env.production' });

const reclient = new Retab();

const automation = await reclient.processors.automations.links.get({
  link_id: "lnk_Xf15nXFpo7mwfGT1aSYo4",
});

console.log(JSON.stringify(automation, null, 2));